src/Command/CreateMySqlCommand.php

```diff
@@ -0,0 +1,208 @@
+
+        $dbName = $helper->ask($input, $output, $question);
+
+        $question = new Question('MySQL port (defaults to "3306"): ', '3306');
+        $question->setValidator(function ($answer) {
+            if (! is_numeric($answer) || $answer === 0) {
+                throw new \RuntimeException(
+                    'Invalid MySQL port'
+                );
+            }
+
+            return $answer;
+        });
+        $question->setMaxAttempts(2);
+
+        $dbPort = $helper->ask($input, $output, $question);
+
+        $question = new Question('MySQL root password (defaults to ""): ', '');
+
+        $mysqlRoot = $helper->ask($input, $output, $question);
+
+        $question = new ConfirmationQuestion('Do you want to create antoher MySQL user? (y/n) ', false);
+
+        $answer = $helper->ask($input, $output, $question);
+
+        if ($answer) {
+            $question = new Question('MySQL user: ', '');
+            $question->setValidator(function ($answer) {
+                if (! is_string($answer) || strlen($answer) === 0) {
+                    throw new \RuntimeException(
+                        'Invalid user'
+                    );
+                }
+
+                return $answer;
+            });
+            $question->setMaxAttempts(2);
+
+            $userName = $helper->ask($input, $output, $question);
+
+            $question = new Question('MySQL password: ', '');
+            $question->setValidator(function ($answer) {
+                if (! is_string($answer) || strlen($answer) === 0) {
+                    throw new \RuntimeException(
+                        'Invalid password'
+                    );
+                }
+
+                return $answer;
+            });
+            $question->setMaxAttempts(2);
+
+            $password = $helper->ask($input, $output, $question);
+        }
+
+        $question = new Question('Mount docker-entrypoint-initdb.d (optional): ', null);
+
+        $initDb = $helper->ask($input, $output, $question);
+
+        $question = <<<EOT
+        
+####################################################
+
+Setup will be done with the following configuration:
+
+Service name: $serviceName
+Image: $image
+Port: $dbPort
+Database name: $dbName
+Root password: $mysqlRoot
+
+EOT;
+        if ($initDb) {
+            $question .= "docker-entrypoint-initdb.d: $initDb\n";
+        }
+
+        if ($answer) {
+            $question .= "User name: $userName\nPassword: $password\n";
+        }
+
+        $question .= "\nAre those settings correct? (y/n): ";
+
+        $confirmation = new ConfirmationQuestion($question, false);
+
+        if (! $helper->ask($input, $output, $confirmation)) {
+            $output->writeln('Aborted');
+
+            return 0;
+        }
+
+        $config = [
+            'services' => [
+                $serviceName => [
+                    'image' => $image,
+                    'ports' => [
+                        "$dbPort:3306",
```

Comment:
Doesn't hurt and helps modifying later, if needed. We should always assume the end-user has no knowledge about docker.